blazor/rich-text-editor/exec-command.md

```diff
@@ -26,7 +26,7 @@ The `ExecuteCommand` methods support following HTML editor commands.
 <td><p>Bold</p></td>
 <td><p>Make the selected content bold.</p></td>
 <td>
-{% highlight cshtml %} 
```

Comment:
I found there are lot of changes in the column tags for removing extra space. I want to know that, Will removing the space fix any issues?

If not, clear these unwanted changes and send the PR again.